### ISSUE TITLE
Spyder 4.1.0, 4.1.1 hangs/freezes when simply pasting a long url in editor

### ISSUE THREAD

**Issue by smisra4** [spyder-ide/spyder#11887]
### Issue Report Checklist

* [done] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [done ] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [done] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [na] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [done] Tried basic troubleshooting (if a bug/error)
    * [done] Restarted Spyder
    * [done] Reset preferences with ``spyder --reset``
    * [done] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [done] Tried the other applicable steps from the Troubleshooting Guide
* [done] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description

A line like the following in spyder editor causes it to crash.
This started in 4.1.0 and 4.1.1. This was fine till 3.3.6.

[spyder4_fail.txt](https://github.com/spyder-ide/spyder/files/4357111/spyder4_fail.txt)

### What steps reproduce the problem?

1. Just copy and paste the line in above file in spyder 4.1.0 or 4.1.1 editor
2. The issue still happens if the line is commented out.
3. If this is put into a file, the spyder hangs while opening the file.
    Just try opening the file in spyder.
 

### What is the expected output? What do you see instead?
NA,  crash, hang

### Paste Traceback/Error Below (if applicable)

```python-traceback

NA, Hang, Freeze
```

## Versions

* Spyder version: 4.1.1
* Python version: 3.7.6
* Qt version: 5.9.7
* PyQt version: 5.9.2
* Operating System name/version:  Windows 10

### Dependencies

# Mandatory:
atomicwrites >=1.2.0           :  1.3.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.3.0 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.13.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.2 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=0.25                  :  2.4.4 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.9 (OK)
qdarkstyle >=2.8               :  2.8 (OK)
qtawesome >=0.5.7              :  0.7.0 (OK)
qtconsole >=4.6.0              :  4.7.1 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.3 (OK)
sphinx >=0.6.6                 :  2.4.0 (OK)
spyder_kernels >=1.9.0;<1.10.0 :  1.9.0 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  18.1.1 (OK)

# Optional:
cython >=0.21                  :  None (OK)
matplotlib >=2.0.0             :  3.1.3 (OK)
numpy >=1.7                    :  1.18.1 (OK)
pandas >=0.13.1                :  1.0.2 (OK)
scipy >=0.17.0                 :  1.4.1 (OK)
sympy >=0.7.3                  :  None (OK)

**Comment by ccordoba12** [comment]
The bug is valid, but I don't think we're going to solve it soon. If you want to edit an string with almost 400k characters, I think you need to use another editor/IDE.

**Comment by smisra4** [comment]
Okay. It was a file URL in one of my programs. 
Interestingly, Spyder till version 3.3.6 handled it without a problem.

**Comment by ccordoba12** [comment]
> Okay. It was a file URL in one of my programs.

I don't think it is. It looks like a base64 encoded image.

**Comment by smisra4** [comment]
No, it is actually a dataURL generated through this https://dataurl.sveinbjorn.org/#dataurlmaker
It generates base64 encoded URLs.
